File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/operation/UpdateShopOperationParam.java
package com.cakk.domain.mysql.dto.param.operation;

import java.util.List;

import lombok.Builder;

import com.cakk.domain.mysql.entity.shop.CakeShopOperation;
import com.cakk.domain.mysql.entity.user.User;

@Builder
public record UpdateShopOperationParam(
	List<CakeShopOperation> cakeShopOperations,
	User user,
	Long cakeShopId
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/operation/UpdateShopOperationParam.kt
package com.cakk.domain.mysql.dto.param.operation

import com.cakk.domain.mysql.entity.shop.CakeShopOperation
import com.cakk.domain.mysql.entity.user.User

data class UpdateShopOperationParam(
	val cakeShopOperations: List<CakeShopOperation>,
	val user: User,
	val cakeShopId: Long
)
